app/jobs/ingest_job.rb

```diff
@@ -54,8 +54,8 @@ def perform(model_params:, background_job_result:, signed_ids: {}, globus_ids: {
     StageGlobus.stage(globus_ids, druid)
 
     # Close the version, which will also start accessioning
-    # start_workflow will be false in the case of reserving a druid
-    Dor::Services::Client.object(druid).version.close(user_versions:) if start_workflow
+    # close will be false in the case of reserving a druid
```

Comment:
for consistency, should `start_workflow` have changed to `accession` instead of `close`?

Reply:
Great question. I initially changed it to "close", but decided to go with "accession" because it aligns with the API.

That being said, all of SDR API would benefit from a variable name refactor.